packages/react-tabs/src/components/Tab/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './Tab.types';
 export * from './renderTab';
 export * from './useTab';
 export * from './useTabStyles';
+export * from './useTabAnimatedIndicator';
```

Comment:
This hook probably should be a part of `useTabStyles`, we don't want to introduce new hooks for users, right?

Reply:
Ling had the same comment above. I'll sync up with Levi and ask.  I think there is some value to separating for the composition model, but can move it down the stack if I need to.